types/ckeditor__ckeditor5-widget/src/widgettypearound/widgettypearound.d.ts

```diff
@@ -0,0 +1,12 @@
+import { Plugin } from '@ckeditor/ckeditor5-core';
+import { Enter } from '@ckeditor/ckeditor5-enter';
+import { Delete } from '@ckeditor/ckeditor5-typing';
+
+export default class WidgetTypeAround extends Plugin {
+    static readonly pluginName: 'WidgetTypeAround';
+    static readonly requires: [typeof Enter, typeof Delete];
+    init(): void;
+    destroy(): void;
+}
+
+export {};
```

Comment:
```suggestion
```
why do you need this? you've explicit export here

Reply:
Fixed. It used to be there cause I had to mock up `ckeditor/typing`.